setuptools/_distutils/tests/test_util.py

```diff
@@ -33,7 +33,7 @@ def environment(monkeypatch):
     monkeypatch.setattr(os.path, 'join', os.path.join)
     monkeypatch.setattr(os.path, 'isabs', os.path.isabs)
     monkeypatch.setattr(os.path, 'splitdrive', os.path.splitdrive)
-    monkeypatch.setattr(sysconfig, '_config_vars', copy(sysconfig._config_vars))
+    monkeypatch.setattr(sysconfig, 'get_config_vars', sysconfig.get_config_vars)
```

Comment:
Changes in `setuptools/_distutils/tests/test_util.py` are overwritten every time `setuptools` syncs up with `pypa/distutils`, so probably they have to be proposed in the `pypa/distutils` repository first.

Reply:
Thanks! I can go make the changes upstream to the right place.